docs/stack/client/querying.md

```diff
@@ -0,0 +1,124 @@
+# Query, Observables & Subscription
+
+In this guide we will describe how to query and subscribe to the subgraph that contains the index of DAOstack data.
+
+## Query
+These are the graphQL queries sent to graphnode to fetch DAOstack data from the subgraph.
+
+Following are the two ways to use client to query the subgraph:
+
+### Standard queries
```

Comment:
Can we call these just "Entity methods"?  They aren't really queries at all, rather are javascript methods on our client entities.  It just happens that the filter params on _some_ of those methods happen to be consistent with GraphQL.